Webserver.ino

```diff
@@ -65,48 +73,67 @@ void handle_NotFound()
 String SendHTML()
 {
   char buf[128];
-  
+
   String ptr = "<!DOCTYPE html> <html>\n";
-  ptr +="<head><meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0, user-scalable=no\">\n";
+  ptr += "<head><meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0, user-scalable=no\">\n";
+
+#if defined(ESP8266)
+#pragma message "ESP8266 stuff happening!"
```

Comment:
also here there message is a bit distracting

Reply:
looks resolved now